source/Calamari/Kubernetes/ResourceStatus/ResourceStatusReportWrapper.cs

```diff
@@ -59,8 +59,10 @@ public bool IsEnabled(ScriptSyntax syntax)
             Dictionary<string, string> environmentVars)
         {
             var customKubectlExecutable = variables.Get(SpecialVariables.CustomKubectlExecutable);
-            var deploymentTimeoutSeconds = variables.GetInt32(SpecialVariables.DeploymentTimeout) ?? 100;
-            var stabilizationTimeoutSeconds = variables.GetInt32(SpecialVariables.StabilizationTimeout) ?? 10;
+            var deploymentTimeoutSeconds = variables.GetInt32(SpecialVariables.DeploymentTimeout) ?? 0;
+            var stabilizationTimeoutSeconds = variables.GetInt32(SpecialVariables.StabilizationTimeout) ?? 0;
```

Comment:
These defaults were added for the ease of debugging only. They should actually be set from the frontend. When the default values are set from the frontend, the defaults here do not take effect. So they are removed to avoid confusion.